charts/Makefile

```diff
@@ -0,0 +1,44 @@
+HELM ?= helm
+HELM_DOCS_IMAGE ?= jnorwood/helm-docs:v1.11.0
+BUILD_DIR ?= bin
+OUTPUT_DIR ?= $(BUILD_DIR)
+CHART_DIR := $(BUILD_DIR)/gadget
+
+IMAGE_TAG := $(shell git describe --tags --abbrev=0 --match "v*")
```

Comment:
How do other folks handle the chart version of the "latest" app? Or they aren't charts available for "latest" apps?

Reply:
Most of the charts are [living outside](https://github.com/bitnami/charts/tree/main/bitnami) so they perhaps don't care about `latest`. Having said that, one can have dev versions (`1.0.0-dev`) for both chart/app and use `--set image.tag=$IMAGE_TAG` (e.g [cilium](https://github.com/cilium/cilium/blob/0447095e6e6bb877a06efd1d8802f6330c2bf786/contrib/scripts/kind-shell-helpers.sh#L68)) installation and worry about versioning only for releases. What we did now should work fine but in case it gets confusing we can rework it in future?